ui/tests/e2e-1000-genomes/test.js

```diff
@@ -7,22 +7,22 @@ jasmine.getEnv().addReporter({
   }
 });
```

Comment:
Instead of separate directories for 1000 genomes and FHS, can you have:

tests/e2e/1000-genomes.js
tessts/e2e/framingham-heart-study.js